File: app/src/main/java/com/codefororlando/orlandowalkingtours/present/activity/MainActivity.java
package com.codefororlando.orlandowalkingtours.present.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.codefororlando.orlandowalkingtours.R;
import com.codefororlando.orlandowalkingtours.present.base.AppBarActivity;

public class MainActivity extends AppBarActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.main_activity);

        setTitle(R.string.title_tours);
    }
}

File: app/src/main/java/com/codefororlando/orlandowalkingtours/MainActivity.kt
package com.codefororlando.orlandowalkingtours

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.codefororlando.orlandowalkingtours.dashboard.DashboardActivity
import com.crashlytics.android.Crashlytics
import io.fabric.sdk.android.Fabric

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Fabric.with(this, Crashlytics())

        startActivity(Intent(this, DashboardActivity::class.java))
        finish()
    }
}
